back/engines/commercial/analysis/app/lib/analysis/LLM/claude_instant_1.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Analysis
+  module LLM
+    class ClaudeInstant1 < Base
```

Comment:
Since it's likely Claude 2.1 will be better at summarizing (and I assume the API for calling either is the same), maybe we could already implement the various Claude models as subclasses of a `Claude` class that does the bulk of the work? Similar to how it's done for the `Gpt` LLM classes, as subclasses of `OpenAI`.

Not blocking for sure, just if you have the time.

Reply:
I'll add Claude 2.1